docs/reference/query-languages/esql/esql-lookup-join.md

```diff
@@ -151,10 +151,15 @@ To use `LOOKUP JOIN`, the following requirements must be met:
     * `short` and `byte` are compatible with `integer` (all represented as `int`)
     * `float`, `half_float`, and `scaled_float` are compatible with `double` (all represented as `double`)
   * For text fields: You can only use text fields as the join key on the left-hand side of the join and only if they have a `.keyword` subfield
+  * `DATE` and `DATE_NANOS` can only be joined against the exact same type.
 
 To obtain a join key with a compatible type, use a [conversion function](/reference/query-languages/esql/functions-operators/type-conversion-functions.md) if needed.
 
-For a complete list of supported data types and their internal representations, see the [Supported Field Types documentation](/reference/query-languages/esql/limitations.md#_supported_types).
+The list of unsupported fields includes all types not supported by {{esql}} as described in the [Unsupported Field Types documentation](/reference/query-languages/esql/limitations.md#_unsupported_types).
```

Comment:
@craigtaverner I think this section is a little messy now, and technically this should be all part of the bulleted list

I suggest we revamp the **Prerequisites** section and use subheadings, and instead of prose/list items for the supported/unsupported types, perhaps a table would be a good alternative?

Reply:
anyways just food for thought, I think the data type compatibility info should be a standalone section in any case, whether it's under prerequisites or not :)

Reply:
I incorporated your changes. I agree it makes things look more similar with two tables. But I think users might find both places independently, so it is not too bad to have two ways. The generated one is more up-to-date. I did make one fix to your table, because we can join between integers and float now too.